app/scripts/arbeitsangebote/overview/arbeitsangeboteoverview.controller.js

```diff
@@ -301,6 +303,20 @@ angular
           }
         },
         {
+          label: gettext('Arbeitsangeboten löschen'),
```

Comment:
```suggestion
          label: gettext('Arbeitsangebot löschen'),
```
In case of a singular or
```suggestion
          label: gettext('Arbeitsangebote löschen'),
```
In case of plural

Reply:
I almost got it !! it needs to be plural. I have corrected it